services/ui-auth/serverless.yml

```diff
@@ -123,11 +123,19 @@ resources:
         GenerateSecret: false # pragma: allowlist secret
         AllowedOAuthFlows:
           - implicit
+          - code
         AllowedOAuthFlowsUserPoolClient: true
         AllowedOAuthScopes:
           - email
           - openid
           - profile
+        AccessTokenValidity: 2
+        IdTokenValidity: 2
+        RefreshTokenValidity: 5
```

Comment:
Just making sure we set these to desired values before we release
Id Token: "You can set the ID token expiration to any value between 5 minutes and 1 day."
Refresh token: "you can set the application's refresh token expiration to any value between 60 minutes and 10 years." (10 years 😰 )
Access token: "You can set the access token expiration to any value between 5 minutes and 1 day."
source: [Cognito](https://docs.aws.amazon.com/cognito/latest/developerguide/amazon-cognito-user-pools-using-the-access-token.html) and similar pages

Reply:
these are the carts values:
```        AccessTokenValidity: 30
        IdTokenValidity: 30
        RefreshTokenValidity: 24
        TokenValidityUnits: 
          AccessToken: "minutes"
          IdToken: "minutes"
          RefreshToken: "hours"```

Reply:
10 years sounds 👌